### PR TITLE
Add Brightline to us.json

### PR DESCRIPTION
**Pull request by Altonss**
Improve North American coverage status #201 by adding Brightline.

### CODE REVIEW

File: feeds/us.json [hunk 1]
```diff
@@ -12,6 +12,11 @@
             "transitland-atlas-id": "f-9-amtrak~amtrakcalifornia~amtrakcharteredvehicle",
             "fix": true
         },
+        {
```
Comment (jbruechert) [inline feeds/us.json:15]:
```suggestion
        {
            "name": "Brightline",
            "type": "transitland-atlas",
            "transitland-atlas-id": "f-brightline~trails"
        },
        {
```

**Review by jbruechert — CHANGES_REQUESTED** [review]
Thanks!

This also needs the matching planned timetable in addition to the realtime updates.

**Comment by mlundblad** [comment]
I think Brightline is local to Florida, right?
So maybe it should go into a us-fl.json?

**Comment by jbruechert** [comment]
I'm not entirely up to date in their plans, but didn't they habe plans to expand to other areas?

**Comment by mlundblad** [comment]
> I'm not entirely up to date in their plans, but didn't they habe plans to expand to other areas?

Right, there's the Brightline West project: https://en.wikipedia.org/wiki/Brightline_West 
So, I guess us.json might be appropriate, thinking about it!

**Pull request merged**